doc/source/yt4differences.rst

```diff
@@ -0,0 +1,72 @@
+.. _yt4differences:
+
+What's New and Different in yt 4.0?
+===================================
+
+If you are new to yt, welcome!  If you're coming to yt 4.0 from an older
+version, however, there may be a few things in this version that are different
+than what you are used to.  We have tried to build compatibility layers to
+minimize disruption to existing scripts, but necessarily things will be
+different in some ways.
+
+.. contents::
+   :depth: 2
+   :local:
+   :backlinks: none
+
+Background
+----------
+
+In yt-3.0 most user-facing operations on SPH data are produced by interpolating SPH data onto a volume-filling octree mesh. Historically this was easier to implement When support for SPH data was added to yt as it allowed re-using a lot of the existing infrastructure. This had some downsides because the octree was a single, global object, the memory and CPU overhead of smoothing SPH data onto the octree can be prohibitive on particle datasets produced by large simulations. Constructing the octree during the initial indexing phase also required each particle (albeit, in a 64-bit integer) to be present in memory simultaneously for a sorting operation, which was memory prohibitive. Visualizations of slices and projections produced by yt using the default settings are somewhat blocky since by default we use a relatively coarse octree to preserve memory. 
+
+In yt-4.0 this has all changed! Over the past two years, Meagan Lang and Matt Turk implemented a new approach for handling I/O of particle data, based on storing compressed bitmaps containing Morton indices instead of an in-memory octree. This new capability means that the global octree index is now no longer necessary to enable I/O chunking and spatial indexing of particle data in yt.
```

Comment:
can you break up these long lines at <=80 characters? Most editors have a way to automatically do this.

Also I did stuff for the demeshening too ;)

Reply:
ah will do!

oops... that was actually a copy and paste from the YTEP ;)